sources/repo-daemon

```diff
@@ -91,28 +102,41 @@ fi
 ## work block
 echo -e "[II] Starting"
 CHECK_VCS=0
-while true; do
-  sleep "${SLEEPTIME}"
+while ${LOOP}; do
   LOGFILE="$(date --utc +%Y-%m-%d-%H-%M).log"
   echo -e "[II] Created log file '${LOGDIR}/${LOGFILE}'"
+  echo -e "[II] Check repository"
   if [[ ${CHECK_VCS} == ${CHECKVCSTIME} ]]; then
-    /usr/bin/repo-check --nocolor -c "${CONF_FILE}" --vcs &> "${LOGDIR}/${LOGFILE}"
+    /usr/bin/repo-check -s --nocolor -c "${CONF_FILE}" --vcs &> "${LOGDIR}/${LOGFILE}"
     CHECK_VCS=0
   else
-    /usr/bin/repo-check --nocolor -c "${CONF_FILE}" &> "${LOGDIR}/${LOGFILE}"
+    /usr/bin/repo-check -s --nocolor -c "${CONF_FILE}" &> "${LOGDIR}/${LOGFILE}"
     CHECK_VCS=$((${CHECK_VCS}+1))
   fi
   [[ ${CHECK_VCS} == 65535 ]] && CHECK_VCS=0
+  echo -e "[II] Update repository"
+  /usr/bin/repo-update --nocolor -c "${CONF_FILE}" &>> "${LOGDIR}/${LOGFILE}"
```

Comment:
what is the reason to separate this call? For example some changes may be made inside `repo-check` and it means that these changes should be done here as well...